### PR TITLE
Use in-memory NamespaceStore implementation for test cases.

### PR DESCRIPTION
**Pull request by anwar6953**
Example of failing test case: http://builds.cask.co/browse/CDAP-DUT3185-7

https://issues.cask.co/browse/CDAP-4346
http://builds.cask.co/browse/CDAP-DUT3262-2

### CODE REVIEW

**Comment by gokulavasan** [comment]
@anwar6953 Are the tests flaky?

**Comment by anwar6953** [comment]
@gokulavasan 
No, there was a recent change that went in that makes them unconditionally fail:
https://github.com/caskdata/cdap/pull/4541

File: cdap-data-fabric/src/main/java/co/cask/cdap/store/InMemoryNamespaceStore.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import javax.annotation.Nullable;
+
+/**
+ * In-memory implementation of {@link NamespaceStore}.
+ */
+public class InMemoryNamespaceStore implements NamespaceStore {
```
Comment (chtyim) [inline cdap-data-fabric/src/main/java/co/cask/cdap/store/InMemoryNamespaceStore.java]: This class is used in unit-test only. Should move it to src/test directory.

**Comment by chtyim** [comment]
Just one comment about the location of the test class. Otherwise LGTM.

**Comment by anwar6953** [comment]
Moved to same package as the test cases and to path:
/cdap/cdap-data-fabric-tests/src/test/java/co/cask/cdap/data/stream/InMemoryNamespaceStore.java

**Pull request merged**